department-of-veterans-affairs/caseflow
# Add Nod Date Update Timeline Component to Case Timeline

Resolves [CASEFLOW-201](https://vajira.max.gov/browse/CASEFLOW-201)

### Description
Add NOD Date Update Component to Case Timeline

### Acceptance Criteria
- [ ] Nod Date Update details display in timeline (user, old NOD, new NOD, change reason) in correctly sorted order

### Testing Plan
- Switch to a used that can edit nod date such as 
- Go to an AMA appeal e.g. 000100013
- Edit the NOD Date if no update exists in the timeline
- Verify that the update appears in the timeline after reloading the page

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="860" alt="Screen Shot 2021-02-02 at 4 27 15 PM" src="https://user-images.githubusercontent.com/68879427/106816998-1660c680-662b-11eb-8f6b-fac4b7056bcc.png">|<img width="1183" alt="Screen Shot 2021-02-02 at 3 46 16 PM" src="https://user-images.githubusercontent.com/68879427/106817011-1b257a80-662b-11eb-9052-d1f8d1a53379.png">

### Storybook Story
*For Frontend (Presentational) Components*
* [ ] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the component file
